android/app/src/test/java/org/chrome_on_device_ml/ExampleUnitTest.java

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2020. All Rights Reserved.
```

Comment:
just remove this file

Reply:
Done. Should I remove [this file](https://github.com/mehrdadh/chrome-on-device-ml/blob/app/android/app/src/androidTest/java/org/chrome_on_device_ml/ExampleInstrumentedTest.java) as well?